src/hats_import/pipeline_resume_plan.py

```diff
@@ -280,11 +287,20 @@ def get_formatted_stage_name(stage_name) -> str:
     if stage_name is None or len(stage_name) == 0:
         stage_name = "progress"
 
-    return f"{stage_name.capitalize(): <10}"
+    if pipeline_name is None or len(pipeline_name) == 0:
```

Comment:
```suggestion
    if not pipeline_name:
```

seems more Pythonic to me.